packages/tooling/fast-tooling-react/src/form/custom-controls/css/color-picker-component/index.ts

```diff
@@ -0,0 +1,31 @@
+import { attr, customElement } from "@microsoft/fast-element";
+import { ColorPicker } from "./color-picker";
+import { ColorPickerTemplate as template } from "./color-picker.template";
+
+import { ColorPickerStyles as styles } from "./color-picker.styles";
+
+/**
+ *
+ * @public
+ * @remarks
+ * HTML Element: \<color-picker\>
+ *
+ * {@link https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot/delegatesFocus | delegatesFocus}
+ */
+@customElement({
+    name: "color-picker",
+    template,
+    styles,
+    shadowOptions: {
+        delegatesFocus: true,
+    },
+})
+export class FASTColorPicker extends ColorPicker {
+
+    /**
+     * @internal
+     */
+    public connectedCallback() {
+        super.connectedCallback();
+    }
```

Comment:
If we aren't doing any work here do we need to super anything? This can likely be removed.

Reply:
This is necessary. This is how connectedCallback in color-picker.ts is executed.